examples/non_cartesian_reconstruction_stack_3d.py

```diff
@@ -30,7 +30,7 @@
 
 # Loading input data
 image = get_sample_data('3d-pmri')
-image = pysap.Image(data=np.sum(image.data, axis=0))
+image = pysap.Image(data=np.sqrt(np.sum(np.abs(image.data)**2, axis=0)))
```

Comment:
Yes we could although I am not sure that for tutorial/example purposes it is  great since we compute the SOS as it has been suggested by [Roemer](https://onlinelibrary.wiley.com/doi/epdf/10.1002/mrm.1910160203)

Reply:
what I am saying is that this formula is I think equivalent to :
`np.linalg.norm(image.data, axis=0)` no ?

Reply:
Yes it is, but having it explicitly make it more explicitly

Reply:
ok sure